CMakeLists.txt

```diff
@@ -19,7 +19,8 @@
 #
 
 cmake_minimum_required(VERSION 3.10)
-if(${CMAKE_VERSION} VERSION_GREATER_EQUAL 3.13)
+message(STATUS "The CMake identification is ${CMAKE_VERSION}")
```

Comment:
why not just "CMake version: ${CMAKE_VERSION}"

Reply:
The next line in the CMake log is `-- The C compiler identification is GNU 7.5.0`, but I wouldn't mind using your formulation.

Reply:
formulation was changed